### ISSUE TITLE
Memory leaks

### ISSUE THREAD

**Issue by djbowman1** [Sdcross/MPX-OS#1]
Fix memory leaks @Sdcross

**Comment by Sdcross** [comment]
I'm not seeing any more on my side, do you know of some examples?

**Comment by djbowman1** [comment]
I just remember from testing before our final presentatoin we bumped up
memory alloc to a much higher number to avoid any issues

**Comment by Sdcross** [comment]
You're right we did, but I thought we also put a fix in to keep it from leaking? Or was it only a patch for most of them?

**Comment by djbowman1** [comment]
It was only for a big portion of it, if im not mistaken we were still
leaking a bit here and there

**Comment by Sdcross** [comment]
We may have been.
